com.unity.render-pipelines.universal/Editor/UniversalRenderPipelineAssetEditor.cs

```diff
@@ -295,6 +293,80 @@ void DrawQualitySettings()
             EditorGUILayout.EndFoldoutHeaderGroup();
         }
 
+        protected override void OnHeaderGUI()
+        {
+            // Default Header
+            base.OnHeaderGUI();
+
+            // New button in header to assign asset
+            EditorGUILayout.BeginVertical();
+
+            if (GUILayout.Button("Assign to Project Settings - Graphics"))
+            {
+                var asset = (RenderPipelineAsset)target;
+                var pipelineAsset = GraphicsSettings.defaultRenderPipeline;
+                string message = "";
+                string buttonText = "";
+                if (pipelineAsset == null)
+                {
+                    message = $"This will assign {asset.name} to the current Graphics Settings.\nRight now nothing is assigned. Do you want to assign this asset?";
+                    buttonText = "Assign";
+                }
+                else
+                {
+                    message = $"This will assign {asset.name} to the current Graphics Settings.\nRight now {pipelineAsset.name} is assigned. Do you want to replace it?";
+                    buttonText = "Replace";
+                }
```

Comment:
You can extract this to it's own method:
```
bool PromptAssingTo(string assetName, string title, RenderPipelineAsset asset)
        {
            if (asset == null)
            {
                return EditorUtility.DisplayDialog(
                    $"Assign to {title}",
                    $"This will assign {assetName} to the current {title}.\nRight now nothing is assigned. Do you want to assign this asset?",
                    "Assign", "Cancel");
            }
            
            return EditorUtility.DisplayDialog($"Assign to {title}", $"This will assign {assetName} to the current {title}.\nRight now {asset.name} is assigned. Do you want to replace it?", "Replace", "Cancel");
        }
```

Then you can simply:

`if (GraphicsSettings.defaultRenderPipeline != asset && PromptAssingTo(asset.name, "Graphics Setting", pipelineAsset))`

and 

`if (QualitySettings.renderPipeline != asset && PromptAssingTo(asset.name, "Quality Settings", qualityAsset))`

avoiding also the string operations if the conditions are not matched.